bigframes/_config/__init__.py

```diff
@@ -29,44 +32,77 @@ class Options:
     """Global options affecting BigQuery DataFrames behavior."""
 
     def __init__(self):
+        self._local = threading.local()
+        self._local.display_options = display_options.DisplayOptions()
+        self._local.sampling_options = sampling_options.SamplingOptions()
+        self._local.compute_options = compute_options.ComputeOptions()
+
+        # BigQuery options are special because they can only be set once per
+        # session, so we need an indicator as to whether we are using the
+        # thread-local session or the global session.
+        self._local.is_bigquery_thread_local = False
         self._bigquery_options = bigquery_options.BigQueryOptions()
-        self._display_options = display_options.DisplayOptions()
-        self._sampling_options = sampling_options.SamplingOptions()
-        self._compute_options = compute_options.ComputeOptions()
+        self._local.bigquery_options = None
+
+    def _init_biquery_thread_local(self):
```

Comment:
typo: bigquery

Reply:
Thanks. Fixed in [4fa1a92](https://github.com/googleapis/python-bigquery-dataframes/pull/652/commits/4fa1a924782947aa343642e6f31600d969e62392)